apps/tx-builder/src/scripts/deploy-test-basic-contract.js

```diff
@@ -0,0 +1,36 @@
+const RINKEBY_ETHERSCAN_URL = 'https://rinkeby.etherscan.io'
+const TEST_CONTRACT_NAME = 'BasicTypesTestContract'
+
+async function main() {
```

Comment:
would be great to add the deploy plugin sometime: https://github.com/wighawag/hardhat-deploy/tree/master

It'd deploy the contract to the same address across all the networks